spring-kafka/src/main/java/org/springframework/kafka/listener/DeadLetterPublishingRecoverer.java

```diff
@@ -45,6 +46,7 @@
 import org.springframework.kafka.KafkaException;
 import org.springframework.kafka.core.KafkaOperations;
 import org.springframework.kafka.core.ProducerFactory;
+import org.springframework.kafka.listener.DeadLetterPublishingRecoverer.HeaderNames.HeadersToAdd;
```

Comment:
Just to confirm. There is no policy to avoid imports from the class that is declared in the same file, is there? There are two places that use `HeadersToAdd` rather than `HeaderNames.HeadersToAdd`, which causes this import statement to appear.